benefits/static/css/styles.css

```diff
@@ -2,7 +2,7 @@
 
 body {
   font-weight: 400;
-  font-size: 16px;
+  font-size: 18px;
```

Comment:
This also affects the Eligibility Start page. I would declare this font-size 18 somewhere where it _only_ touches the Help page
<img width="1303" alt="image" src="https://user-images.githubusercontent.com/3673236/164304676-633aea55-62e9-4f58-a7ac-ddde62e11a5a.png">

Reply:
Ahh! Thanks for catching these. I forgot to go back through the other pages to check on this type of thing. This should help me remember. 🙏